<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFacilityDialog</class>
 <widget class="QDialog" name="AddFacilityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Facility</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameField">
       <property name="placeholderText">
        <string>Enter Facility Name...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string>X Coordinate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="xSpinBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="yLabel">
       <property name="text">
        <string>Y Coordinate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="ySpinBox"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="acuteCareLabel">
       <property name="text">
        <string>Acute Care</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="acuteCareSpinBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>16777215</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="complexContinuingCareLabel">
       <property name="text">
        <string>Complex Continuing Care</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="complexContinuingCareSpinBox">
       <property name="maximum">
        <number>16777215</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="longTermCareLabel">
       <property name="text">
        <string>Long Term Care</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="longTermCareSpinBox">
       <property name="maximum">
        <number>16777215</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="regionLabel">
       <property name="text">
        <string>Region</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="regionComboBox">
       <item>
        <property name="text">
         <string>Renfrew County</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>North Lanark/North Grenville</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ottawa West</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ottawa Center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ottawa East</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eastern Counties</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
